### DISCUSSION TITLE
How can I set Filebrowser up with Authelia?

### DISCUSSION THREAD

**Discussion by poperigby** [filebrowser/filebrowser#2911] [Q&A]
I'm trying to set up header authentication, so users that are logged into Authelia are automatically logged into File Browser, and new users will automatically be created in File Browser. Is this possible?

I currently have this in my settings file:

```json
{
    "port": 80,
    "baseURL": "",
    "address": "",
    "log": "stdout",
    "database": "/database.db",
    "root": "/srv",
    "auth": {
        "method": "proxy",
        "header": "Remote-User"
    }
}
```

When I login to Authelia, and then go to my File Browser URL, it makes me login to File Browser still, and my Authelia credentials won't work.

I'm using Caddy for my reverse proxy, and this in the configuration entry for File Browser:

```
drive.haddock.cc {
    import auth
    reverse_proxy filebrowser-pod:80
}
```

I'm running this all in Podman on a shared network.

**Comment by thorinmesser** [comment]
I think the Authelia username must be set up as a user in File Browser. Perhaps you can script that as part of creating users in Authelia? This looks like a promising hint to start depending on your setup: https://filebrowser.org/cli/filebrowser-users-add

I use docker containers and I expect that could get complicated, trying to run a command in the shell of one container from an event in another container, but I'm not exactly a docker expert so I could be wrong.
